src/main/webapp/scripts/civic.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Extract the government officials from the civic API response.
+ */
+function extractOfficials(civicObj) {
+  let officials = new Object();
+  const levels = ["administrativeArea1", "administrativeArea2", "country", "international", "locality", "regional", "special", "subLocality1", "subLocality2"];
```

Comment:
[Nit] Use CONSTANT_CASE for constant variables.

Reply:
This isn't a module-local (top-level) constant.